docs/connectors/sources/pubsub.md

```diff
@@ -0,0 +1,150 @@
+# Google Pub/Sub input connector
```

Comment:
do we have some page that lists all all available connectors? 
this should be linked from that page.

Reply:
It is linked from the sidebar and also from the list of connectors (https://www.feldera.com/docs/connectors/sources/) (it's auto-generated).